Parsers/ASimNetworkSession/ARM/vimNetworkSessionPaloAltoCEF/vimNetworkSessionPaloAltoCEF.json

```diff
@@ -2,33 +2,40 @@
allow\" and eventresult==\"Success\") or (eventresult==\"Failure\"))\n| parse AdditionalExtensions with \"PanOSPacketsReceived=\" DstPackets:long \",PanOSPacketsSent=\" SrcPackets:long \",start=\"EventStartTime \",reason=\"*\n| project-rename \n      EventProductVersion=DeviceVersion // Not Documented\n    , Dvc=DeviceName   \n    , NetworkApplicationProtocol=ApplicationProtocol\n    , SrcZone=DeviceCustomString4 \n    , DstZone=DeviceCustomString5\n    , NetworkRuleName=DeviceCustomString1\n    , SrcUsername=SourceUserName \n    , DstUsername=DestinationUserName \n    , EventOriginalSeverity=LogSeverity // not documented\n    , SrcNatIpAddr=SourceTranslatedAddress\n    , DstNatIpAddr=DestinationTranslatedAddress\n    , PaloAltoFlags=FlexString1 // Flags\n| extend\n  EventVendor=\"Palo Alto\"\n    ,EventProduct=\"PanOS\"\n    , DstBytes=tolong(ReceivedBytes) \n    , SrcBytes=tolong(SentBytes)      \n    , NetworkPackets=tolong(DeviceCustomNumber2) \n    , NetworkProtocol=toupper(Protocol)\n    , NetworkBytes=tolong(FlexNumber1)\n    , SrcUsernameType=case(isempty(SrcUsername), \"\", SrcUsername contains \"@\", \"UPN\", \"Simple\")\n    , DstUsernameType=case(isempty(DstUsername), \"\", DstUsername contains \"@\", \"UPN\", \"Simple\")\n    , EventType=\"NetworkSession\"\n    , EventCount=toint(1)\n    , EventResult=case(DeviceAction==\"allow\",\"Success\",\"Failure\")\n    , NetworkSessionId=tostring(DeviceCustomNumber1)\n    , NetworkDuration=toint(1000*DeviceCustomNumber3)\n    , EventSchemaVersion=\"0.2.1\"\n    , EventSchema=\"NetworkSession\"\n    , EventSeverity = \"Informational\"\n    , EventStartTime=coalesce(todatetime(EventStartTime), TimeGenerated)\n  | extend hostelements=split(Dvc,'.')\n  | extend DvcHostname=tostring(hostelements[0])\n        , DvcDomain=strcat_array( array_slice(hostelements,1,-1), '.')\n  | extend DvcFQDN = iff(Dvc contains \".\",Dvc,\"\" )\n        , DvcDomainType=iff(Dvc contains \".\",\"FQDN\",\"\" )\n| project-away hostelements\n| lookup Actions on DeviceAction\n// Action post filtering\n| where (array_length(dvcaction)==0 or DvcAction has_any (dvcaction))\n| project-rename\n    DstMacAddr=DestinationMACAddress\n    , SrcMacAddr=SourceMACAddress\n    , DstIpAddr=DestinationIP\n    , DstPortNumber=DestinationPort\n    , DstNatPortNumber=DestinationTranslatedPort\n  , SrcPortNumber=SourcePort\n    , SrcIpAddr=SourceIP\n  , SrcNatPortNumber=SourceTranslatedPort\n  , DvcOutboundInterface=DeviceOutboundInterface\n  , DvcInboundInterface=DeviceInboundInterface\n  , EventMessage=Message\n  , DvcOriginalAction=DeviceAction\n// -- Aliases\n| extend\n  IpAddr = SrcIpAddr,\n  Rule=NetworkRuleName,\n  Dst=DstIpAddr,\n  // Host=DstHostname,\n  User=DstUsername,\n  Duration=NetworkDuration,\n  SessionId=NetworkSessionId,\n  EventEndTime =EventStartTime,\n  Src=SrcIpAddr\n};\nNWParser(starttime, endtime, srcipaddr_has_any_prefix, dstipaddr_has_any_prefix, dstportnumber, hostname_has_any, dvcaction, eventresult, disabled)\n",
```

Comment:
did we change the parser?

Reply:
I checked the history, and we did. In general, the repackaging is there to ensure all YAML updates are in.